ArduPlane/servos.cpp

```diff
@@ -440,6 +440,14 @@ void ParametersG2::FWD_BATT_CMP::update()
 // Apply throttle scale to min and max limits
 void ParametersG2::FWD_BATT_CMP::apply_min_max(int8_t &min_throttle, int8_t &max_throttle) const
 {
+    // cut off throttle if FWD_BAT_IDX battery is below FWD_THR_CUTOFF_V resting voltage
+    if (AP::battery().voltage_resting_estimate(batt_idx) < batt_voltage_throttle_cutoff) {
```

Comment:
```suggestion
    if (is_positive(batt_voltage_throttle_cutoff) && (AP::battery().voltage_resting_estimate(batt_idx) < batt_voltage_throttle_cutoff)) {
```
Just to be sure that some battery bug where it returns less than zero for resting voltage doesn't cut throttle if the feature is not enabled.